src/docs/user/references/Variables.adoc

```diff
@@ -140,6 +140,12 @@ Examples: PA:JSON will accept {"name": "John", "city":"New York"} and empty valu
 * *PA:GLOBAL_FOLDER* : variable whose value is the relative path of a folder in the <<_global_and_user_spaces, Global Data Space>>. At workflow submission, the scheduler server verifies that the folder exists in the global dataspace. Note, the variable value should not end with a slash to avoid the problem of duplicate slash in its usage.
 * *PA:USER_FOLDER* : variable whose value is the relative path of a folder in the <<_global_and_user_spaces, User Data Space>>. At workflow submission, the scheduler server verifies that the folder exists in the user dataspace. Note, the variable value should not end with a slash to avoid the problem of duplicate slash in its usage.
 
+===== Variable Substitution
+Models can use the value of other variables in their definition. +
+For example: +
+The model PA:LIST($var1,$var2,$var3) uses the value of the 3 variables var1, var2 and var3 to build the list of acceptable entries.
```

Comment:
please add back ticks arount example values and eample variable names to improve the presentation